code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm

```diff
@@ -300,6 +300,16 @@
 	time = 120
 	category = CAT_WEAPONRY
 	subcategory = CAT_WEAPON
+	
+/datum/crafting_recipe/m29snub
+	name = "snubnose .44 magnum revolver"
+	result = /obj/item/gun/ballistic/revolver/m29/snub
+	reqs = list(/obj/item/gun/ballistic/revolver/m29,
+				/obj/item/stack/crafting/goodparts = 1)
```

Comment:
```suggestion
				/obj/item/stack/crafting/goodparts = 1,
				)
```

Reply:
The code standard introduced is trailing commas to lists is not optional, the parenthesis marking end of list I am unsure if its a rule yet but it probably will be, and its a little easier to work with so might as well.